### PR TITLE
bat, cgclassify: add Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.nl/common/bat.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+`bat {{pad/naar/bestand}}`
+
+- Voeg verschillende bestanden samen in het doelbestand:
+
+`bat {{bestand1}} {{bestand2}} > {{doelbestand}}`
+
+- Voeg verschillende bestanden toe aan het doelbestand:
+
+`bat {{bestand1}} {{bestand2}} >> {{bestand}}`
```
Comment (leonvsc) [inline pages.nl/common/bat.md:17]:
```suggestion
`bat {{bestand1}} {{bestand2}} >> {{doelbestand}}`
```

File: pages.nl/linux/cgclassify.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# cgclassify
+
+> Verplaats lopende taken naar opgegeven `cgroups`.
+> Meer informatie: <https://manned.org/cgclassify>.
+
+- Verplaats het proces met een specifiek PID naar de controle group student in de CPU hierarchie:
```
Comment (leonvsc) [inline pages.nl/linux/cgclassify.md:6]:
```suggestion
- Verplaats het proces met een specifiek PID naar de controle groep student in de CPU hierarchie:
```

File: pages.nl/linux/cgclassify.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`cgclassify -g {{cpu:student}} {{1234}}`
+
+- Verplaats het proces met een specifiek PID naar de controle groups gebaseerd op het `/etc/cgrules.conf` configuratie bestand:
```
Comment (leonvsc) [inline pages.nl/linux/cgclassify.md:10]:
```suggestion
- Verplaats het proces met een specifiek PID naar de controle groepen gebaseerd op het `/etc/cgrules.conf` configuratie bestand:
```

File: pages.nl/linux/cgclassify.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`cgclassify {{1234}}`
+
+- Verplaats het proces met een specifiek PID naar de controle group student in de CPU hierarchy. Let op: de daemon van de service `cgred` veranderd `cgroups` van de specifieke PID en zijn children niet (gebaseerd op `/etc/cgrules.conf`):
```
Comment (leonvsc) [inline pages.nl/linux/cgclassify.md:14]:
```suggestion
- Verplaats het proces met een specifiek PID naar de controle groep student in de CPU hierarchy. Let op: de daemon van de service `cgred` veranderd `cgroups` van de specifieke PID en zijn onderliggende processen niet (gebaseerd op `/etc/cgrules.conf`):
```

**Review by leonvsc — APPROVED** [review]
Looks good to me with a few suggestions.

**Pull request merged**